quickstarts/android/02-customlogin.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Auth0 Android Quickstarts - 2. Custom Login
+description: This tutorial will show you how to use the Auth0 authentication api in your Android project to create a custom login screen.
+name: Android
+alias:
+  - Android
+  - Login
+  - Custom
+  - Headless
+  - Quickstart
+  - Mobile 
+language:
+  - Android
+hybrid: false
+image: /media/platforms/android.png
+tags:
+  - quickstart
+  - headless
+  - custom
+  - login
+alias:
+  - android
+seo_alias: android
+---
+
+## Android - Custom Login Tutorial
+
+This is a simple, quickstart that will show you how to add the Auth0 login capabilities while using a customized login screen.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+
+### Before Starting
+
+<div class="setup-callback">
+<p>Go to the <a href="${uiAppSettingsURL}">Application Settings</a> section in the Auth0 dashboard and make sure that <b>Allowed Callback URLs</b> contains the following value:</p>
+
+<pre><code>a0${account.clientId}://\*.auth0.com/authorize</pre></code>
+</div>
+
+### 1. Add the Lock dependency
+
+Your first step is to add [Lock](https://github.com/auth0/Lock.Android) into your project, which is basically a library for displaying native UI in your app for logging in and signing up with different social platforms via [auth0](https://auth0.com/).
```

Comment:
I'm not sure on Android, but, at least on iOS, Lock was not needed for this tutorial. In fact, that was one of its aims: not to use the Lock library.